monasca/values.yaml

```diff
@@ -265,6 +265,17 @@ grafana:
   service:
     type: NodePort
     port: 3000
+  keystone:
```

Comment:
does grafana need the project name, domain name, and project domain name? Why did we never have to set it before and still have it work? I thought it was a keystone endpoint and thats it? Teach me!